src/Game/ShipSpawner.cpp

```diff
@@ -578,6 +627,11 @@ glm::vec2 ShipSpawner::find_nearest_non_pirate_ship(std::shared_ptr<Ship> const&
 
 glm::vec2 ShipSpawner::find_nearest_ship(glm::vec2 center_position)
 {
+    if (m_ships.size() == 0)
+    {
+        return { 30.0f, 30.0f };
```

Comment:
Why 30.0f? I suppose its to show that it's nowhere near? It could technically bite us one day, but it's highly improbable. We may just return NaN, but it would require extra handling.